spec/classes/foreman_database_spec.rb

```diff
@@ -63,8 +63,8 @@
       describe 'with seed parameters' do
```

Comment:
Maybe this should be exanded to include all `initial_*` parameters to verify they're set correctly.